src/common/actions/snaps.js

```diff
@@ -12,14 +12,16 @@ export const REMOVE_SNAP = 'REMOVE_SNAP';
 export const REMOVE_SNAP_SUCCESS = 'REMOVE_SNAP_SUCCESS';
 export const REMOVE_SNAP_ERROR = 'REMOVE_SNAP_ERROR';
 
-export function fetchUserSnaps(owner) {
+export function fetchUserSnaps(owner, showFetching = true) {
```

Comment:
New param stops fetching status (i.e. the spinner) from being shown while fetching new snap data. This is useful when polling the service for updates.